### PR TITLE
Community grants section

### PR DESCRIPTION
**Pull request by technophile-04**
## Description

Lol had to play around with background image height but it kinda works on mobile too, we can play more with heights if it doesn't good look  🙌
Demo : 

https://github.com/BuidlGuidl/grants.buidlguidl.com/assets/80153681/9c840bfe-99d4-4271-b23b-9b3078491fb4

### CODE REVIEW

**Review by carletex — CHANGES_REQUESTED** [review]
This is amazing Shiv!!! <333 

Pushed some tweaks to the copy (and some tiny tweaks on spacing).

A couple of things:
1. Could you make the "learn more" button from the hero to go to this section?
2. Could you change the orange patch to this?
orange-patch

If not, it cuts off on mobile. On desktop, you might need to move the image down and "hide the overflow" in the section.

**Comment by technophile-04** [comment]
> Could you change the orange patch to this?

> if not, it cuts off on mobile. On desktop, you might need to move the image down and "hide the overflow" in the section.

Tysm for the image !! Updated at https://github.com/BuidlGuidl/grants.buidlguidl.com/pull/25/commits/f8a4a8cc6c1d8696994f7bff6b2c028fdf88fe7a, could you check once 🙌

>  Could you make the "learn more" button from the hero to go to this section?

Ohh missed it ! Updated at https://github.com/BuidlGuidl/grants.buidlguidl.com/pull/25/commits/6fab426b0544b9dc98cf375dad96f67be6e0a892

File: packages/nextjs/app/_components/CommunityGrant.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import Image from "next/image";
+import Link from "next/link";
+
+export const CommunityGrant = () => {
+  return (
+    <div id="communityGrants" className="bg-secondary">
+      <div className="container max-w-[90%] sm:max-w-md lg:max-w-7xl m-auto xl:pl-18 lg:pl-8 flex flex-col-reverse lg:flex-row gap-5 lg:gap-0 max-h-[30rem]">
```
Comment (Pabl0cks) [inline packages/nextjs/app/_components/CommunityGrant.tsx:7]: Maybe we could add some marging for smaller devices (those where we are showing the full orange shape), something like: `my-6 lg:my-0`

**Review by Pabl0cks — COMMENTED** [review]
Starting to get familiar with the repo, just added a small nitpick, feel free to ignore if you see it better like it is now🙌

File: packages/nextjs/app/_components/CommunityGrant.tsx (hunk 1 above)
Comment (technophile-04) [reply packages/nextjs/app/_components/CommunityGrant.tsx]: Yup I think it makes sense, update at https://github.com/BuidlGuidl/grants.buidlguidl.com/pull/25/commits/7cef5a5f30ffc65ce936aafbfdd5c0773a57c133, Thanks Pablo 🙌

**Review by carletex — APPROVED** [review]
Amaziiiing!!

**Pull request merged**